packages/client/tests/e2e/_utils/docker-compose-clean.yml

```diff
@@ -2,14 +2,25 @@ version: '3.7'
 
 services:
   # not an actual service, just a helper to clean up the workspace
-  clean:
+  pre-clean:
```

Comment:
Question: why are we running it through docker? As far as I can tell, it cleans stuff only in mounts, so we could probably just run same commands on the host. Or am I missing something?

Reply:
On Linux docker, the created files are created with `root` user. I don't think mac users have this inconvenience, though. I could change the user in the container so that files are created with the correct user, but that would bring other issues. So I went for an universal approach.

Reply:
Updated the comment